packages/img/src/getImage.ts

```diff
@@ -10,6 +14,8 @@ const getImage: EndpointInstance<
     { src }: { src: string },
   ): Promise<string> {
     return new Promise(resolve => {
+      // even if we polyfill, on server we don't want to actually wait to resolve the Image
```

Comment:
i don’t think the check you want is “is node” - Image only exists in a browser, so i think the check you want is “isBrowser” - namely, does `window` exist

Reply:
ya but if someone somehow polyfills RN or something I"m okay with them using this. VS on server, even if it's polyfilled it would be silly to do.

Reply:
in RN, window exists.

Reply:
didn't realize that - ok

Reply:
I’m not 100% but I’m pretty sure it’s like a browser environment

Reply:
yeah, I know they don't mock everything and it creates a lot of weird scenarios.